test_chunk_utils.py

```
@@ -0,0 +1,121 @@
+import unittest
+from unittest import mock
```

Ruff (0.8.2)

`unittest.mock` imported but unused

Remove unused import: `unittest.mock`

(F401)